vaccine_feed_ingest/runners/sc/arcgis/normalize.py

```diff
@@ -135,7 +135,10 @@ def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
 def _get_normalized_location(
     site: dict, timestamp: str
 ) -> Optional[schema.NormalizedLocation]:
-    if len(site["attributes"]["loc_name"]) > 256:
+    if site["attributes"] is None:
+        return None
+    name = site["attributes"]["loc_name"]
+    if name is None or len(name) > 256:
         return None
```

Comment:
Could we add an error log for both of these cases?

Reply:
Done! Also fixed a type error in a related location.